lib/TeamcityReporter.js

```diff
@@ -33,10 +33,19 @@ class TeamcityReporter {
     item(err, args) {
         if (!this.currItem.passed) {
             const msg = this.escape(this.currItem.failedAssertions.join(", "));
-            const responseCode = (this.currItem.response && this.currItem.response.responseTime) || "-";
+            const responseCode = this.currItem.response.code;
             const reason = (this.currItem.response && this.currItem.response.reason()) || "-";
             const details = this.escape(`Response code: ${responseCode}, reason: ${reason}`);
-            console.log(`##teamcity[testFailed name='${this.currItem.name}' message='${msg}' details='${msg} - ${details}']`);
+            console.log(`##teamcity[testFailed name='${this.currItem.name}' details='${msg} - ${details}']`);
+            var responseBody;
+			try {
+				responseBody = JSON.stringify(this.currItem.response.json());
+				responseBody = this.escape(`${responseBody}`);
+				console.log(`##teamcity[testFailed responseBody ='${responseBody}'`);
+			}
+			catch (e) {
+				console.log(`##teamcity[testFailed Exception : '${e}']`);
+			}
```

Comment:
Putting the `responseBody` in the Teamcity message seems noisy and undesirable to me and I don't think the catch  block is logging a valid Teamcity message.

Reply:
Well, fair enough. You can ignore this change proposal.